From 4c660cfcc607f51b600393d1944455c7b66b5c39 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Thu, 30 Aug 2018 16:49:25 +0300
Subject: [PATCH 0682/1051] phy: cp110-comphy: add missing comphy
 configurations

This patch adds the following comphy configurations:
	SATA_PORT1@COMPHY0
	SATA_PORT0@COMPHY2
	USB3_PORT1@COMPHY3

These configurations are valid, according to CP110/5
reference manual.

Change-Id: I7fa88d4425f18c5416c91fff6cff4b886c653f54
Signed-off-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/phy/marvell/phy-mvebu-cp110-comphy.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/phy/marvell/phy-mvebu-cp110-comphy.c b/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
index f4d27b8e2ac5..cf720b640187 100644
--- a/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
+++ b/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
@@ -80,6 +80,7 @@ static const struct mvebu_comhy_conf mvebu_comphy_cp110_modes[] = {
 	MVEBU_COMPHY_CONF(0, 1, PHY_MODE_SGMII),
 	MVEBU_COMPHY_CONF(0, 1, PHY_MODE_2500SGMII),
 	MVEBU_COMPHY_CONF(0, 0, PHY_MODE_PCIE),
+	MVEBU_COMPHY_CONF(0, 1, PHY_MODE_SATA),
 	/* lane 1 */
 	MVEBU_COMPHY_CONF(1, 2, PHY_MODE_SGMII),
 	MVEBU_COMPHY_CONF(1, 2, PHY_MODE_2500SGMII),
@@ -92,11 +93,13 @@ static const struct mvebu_comhy_conf mvebu_comphy_cp110_modes[] = {
 	MVEBU_COMPHY_CONF(2, 0, PHY_MODE_10GKR),
 	MVEBU_COMPHY_CONF(2, 0, PHY_MODE_PCIE),
 	MVEBU_COMPHY_CONF(2, 0, PHY_MODE_USB_HOST),
+	MVEBU_COMPHY_CONF(2, 0, PHY_MODE_SATA),
 	/* lane 3 */
 	MVEBU_COMPHY_CONF(3, 1, PHY_MODE_SGMII),
 	MVEBU_COMPHY_CONF(3, 1, PHY_MODE_2500SGMII),
 	MVEBU_COMPHY_CONF(3, 0, PHY_MODE_PCIE),
 	MVEBU_COMPHY_CONF(3, 1, PHY_MODE_SATA),
+	MVEBU_COMPHY_CONF(3, 1, PHY_MODE_USB_HOST),
 	/* lane 4 */
 	MVEBU_COMPHY_CONF(4, 0, PHY_MODE_SGMII),
 	MVEBU_COMPHY_CONF(4, 0, PHY_MODE_2500SGMII),
-- 
2.17.1

